src/lib/viewers/controls/annotations/__tests__/AnnotationsControls-test.tsx

```diff
@@ -1,11 +1,13 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
Is this change necessary?

Reply:
The changes to this file should be unnecessary now since `annotationColor` is optional and defaulted

Reply:
Yep, will revert.